frontend/src/modules/member/components/suggestions/member-merge-similarity.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div class="flex items-center gap-1" :style="{ color: style.color }">
+    <div v-html="style.svg" />
+    <p class="text-xs leading-5 whitespace-nowrap">
+      {{ percentage }}% confidence
+    </p>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { computed } from 'vue';
+
+const props = defineProps<{
+  similarity: number
+}>();
+
+const percentage = computed(() => Math.ceil(props.similarity * 100));
+
+const style = computed<{
+  color: string,
+  svg: string
+}>(() => {
+  if (props.similarity >= 0.8) {
+    return {
+      color: '#059669',
+      svg: `<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M14.1667 2.66699H17.5V17.5003H14.1667V2.66699ZM8.33337 7.5H11.6667V17.5003H8.33337V7.5Z" fill="#059669"/>
+<path d="M2.5 12H5.83333V17.5H2.5V12Z" fill="#059669"/>
+</svg>`,
```

Comment:
Would it be better to create an icon for this?

Reply:
added it to icons